src/esp/sensor/SensorFactory.h

```diff
@@ -0,0 +1,22 @@
+#include "esp/scene/SceneNode.h"
+#include "esp/sensor/CameraSensor.h"
```

Comment:
@vauduong :

This is something I did not catch.
You do not need this header file here.

Reply:
Thank you!